### PR TITLE
[16079] Fix dark mode for communities and messages home

### LINKED ISSUE #16079: Communities Home Page - Implement Dark Mode
The communities home page needs to implemented correctly for Dark mode.

See the designs and implement correctly.
https://www.figma.com/file/h9wo4GipgZURbqqr1vShFN/Communities-for-Mobile?type=design&node-id=5040-188662&t=dyTuecNWxkV9KhMs-4

### PR DESCRIPTION
**Pull request by ibrkhalil**
fixes #16079 

### Summary
Removes the dark background on communities home by adding an `overlay-color` on it's blur view.

After:
image

status: ready

### CODE REVIEW

File: src/status_im2/contexts/chat/home/view.cljs [hunk 1]
```diff
@@ -132,9 +132,10 @@
        [chats selected-tab top])
      [rn/view {:style (style/blur-container top)}
       [blur/view
-       {:blur-amount (if platform/ios? 20 10)
-        :blur-type   (if (colors/dark?) :dark (if platform/ios? :light :xlight))
-        :style       style/blur}]
+       {:blur-amount   (if platform/ios? 20 10)
+        :blur-type     (if (colors/dark?) :dark (if platform/ios? :light :xlight))
+        :overlay-color (if (colors/dark?) colors/neutral-95-opa-70 colors/white-opa-70)
```
Comment (ulisesmac) [inline src/status_im2/contexts/chat/home/view.cljs]: We could use `theme/theme-value` for `:blur-type`  and `colors/theme-colors` for `:overlay-color`

**Review by ulisesmac — APPROVED** [review]
LGTM :+1:

File: src/status_im2/contexts/chat/home/view.cljs (hunk 1 above)
Comment (ibrkhalil) [reply src/status_im2/contexts/chat/home/view.cljs]: Done

**Review by ilmotta — APPROVED** [review]
For PRs with UI fixes, I think having screenshots of a *before vs after* PR is helpful for reviewers :) The original issue just points the correct the way (the after).

**Comment by ibrkhalil** [comment]
> For PRs with UI fixes, I think having screenshots of a _before vs after_ PR is helpful for reviewers :) The original issue just points the correct the way (the after).

I am OOO now, Would this one work? It's from my branch
image

**Comment by ilmotta** [comment]
Thanks @ibrkhalil. No worries if you're OOO.

For the screenshots, I mean a before/after showing the same screen with the same data, not just the branch's screenshot and a different one in Figma. That to me is like :candy: :100:, the sweet spot ;)

----

Btw, if you share high-res screenshots, very common in today's screens, Github displays horrendously huge images, like below. It's a problem in many of our issues and PRs (not just from your last comment) and it makes navigating in the comment section sort of a frustrating experience. The fix is to use the `img` tag and specify a width and/or height.

This is how it looks on my big 1440p screen. Notice the screenshot size compared to the browser's URL bar, it's bizarre.

**Comment by VolodLytvynenko** [comment]
Hi @ibrkhalil Could you rebase the current PR one more time, please? One critical fix https://github.com/status-im/status-mobile/commit/a7ddda40c4667db4f4c4aab50f330831e8274d0c  is merged

**Comment by VladimrLitvinenko** [comment]
Hi @ibrkhalil thank you for PR. Take a look please, one issue is found:
## ISSUE 1: The dark background showed instead of light after the dark mode is selected


### Steps to reproduce:
1. Set the app to light mode.
2. Log out of the app.
3. Log back
4. Set the app to dark mode.
5. Go to the "Communities" or "Messages" page.

### Actual result:
A dark background is shown on the "Communities" or "Messages" page
image

### Expected result:
After setting the app to dark mode, the background on the "Communities" or "Messages" page should be light

### Notes:
The same issue occurs when switching from dark mode to light mode. The background does not update accordingly after the mode change.

### Devices:
- Real device: Pixel 7a, Android 13
- Real device: iPhone 11 Pro max, IOS 16
- Android studio: Pixel XL API 29

**Comment by VolodLytvynenko** [comment]
@ibrkhalil could you please take a look at the integration failed tests https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-16382/14/consoleText

**Comment by ibrkhalil** [comment]
> @ibrkhalil could you please take a look at the integration failed tests https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-16382/14/consoleText

Done

**Comment by VolodLytvynenko** [comment]
> > @ibrkhalil could you please take a look at the integration failed tests https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-16382/14/consoleText
> 
> Done

Thank you! @ibrkhalil let me know when https://github.com/status-im/status-mobile/pull/16382#issuecomment-1614547270 is fixed and  PR is ready for tests

**Comment by VolodLytvynenko** [comment]
Hi @ibrkhalil Thank you for fixing the integration tests and for the PR. I apologize for the invalid issue that was initially reported

One issue is found:

## ISSUE 1: The dark background is still shown for IOS devices.
### Actual result
The dark background is still being shown on iOS devices.
image


### Expected result:
The background should be removed

### Device:
- iPhone 11 Pro max, IOS 16

**Comment by ibrkhalil** [comment]
> Hi @ibrkhalil Thank you for fixing the integration tests and for the PR. I apologize for the invalid issue that was initially reported
> 
> One issue is found:
> 
> ## ISSUE 1: The dark background is still shown for IOS devices.
> ### Actual result
> The dark background is still being shown on iOS devices. image
> 
> ### Expected result:
> The background should be removed
> 
> ### Device:
> * iPhone 11 Pro max, IOS 16

Kindly check again

**Comment by jakubgs** [comment]
Not sure why my review was requested for a purely Clojure change.

**Comment by VolodLytvynenko** [comment]
## ISSUE 2: [Android] Shining effect on elements for Android Devices on dark and light mode

### Actual result:
image
image

### Expected result:
image
The elements are without shining effect

### Devices:
- Real Android: Pixel 7a, Android 13
- Android studio: Pixel XL API 29

**Comment by VolodLytvynenko** [comment]
> Hi @ibrkhalil Thank you for fixing the integration tests and for the PR. I apologize for the invalid issue that was initially reported
> 
> One issue is found:
> 
> ## ISSUE 1: The dark background is still shown for IOS devices.
> ### Actual result
> The dark background is still being shown on iOS devices. image
> 
> ### Expected result:
> The background should be removed
> 
> ### Device:
> * iPhone 11 Pro max, IOS 16

@ibrkhalil issue still not fixed.
How it shown on IOS in latest PR build on dark mode. The background is still highlighted as a separate element on IOS
image

**Comment by VolodLytvynenko** [comment]
@ibrkhalil One question: Should the background be fixed in scope of when PR if the drawer is shown?

Let me know if it is not in the scope of this PR, I will create a separate issue for this
## ISSUE 3: [Android] The "messages" and "communities" background does not matched to design on dark and light mode after the drawer is shown

### Steps to reproduce:
1. Open the drawer on the "messages" and "communities" page (example: tap create chat [+] button

### Actual result
image
image

### Expected result:
image

https://www.figma.com/file/7KIYbhoqNGAIFonE0w9TDz/Messages-for-Mobile?type=design&node-id=1694-378798&mode=design&t=dPITvqtKAnOckins-0

**Comment by ibrkhalil** [comment]
I had to apply a background color when bottom sheet is opened to fix the issue the happened with blur background opacity when bottom sheet is open.

**Comment by VolodLytvynenko** [comment]
Thank you for the fixes @ibrkhalil . No issues from my side. PR is ready to be merged

**Pull request merged**